Sources/SwiftKafka/KafkaTopicConfig.swift

```diff
@@ -37,7 +37,8 @@ public struct KafkaTopicConfig: Hashable, Equatable {
         }
 
         deinit {
-            rd_kafka_topic_conf_destroy(pointer)
+            // rd_kafka_topic_conf_destroy(pointer)
```

Comment:
We are running into an issue here:

The `pointer` to the `rd_kafka_topic_conf_t` gets automatically freed by `rd_kafka_topic_new`. This leads us to a situation where we have a double-freeing of this pointer. However, when the `KafkaTopicConfig` is created without passing it to `rd_kafka_topic_new`, we still need to deallocate it here.

A solution could be checking if the pointer has been freed or not. Though I don't know whether this is possible.